std/assembly/array.ts

```diff
@@ -153,6 +153,30 @@ export class Array<T> {
     return newLength;
   }
 
+  concat(items: Array<T>): Array<T> {
+    assert(this != null);
+    if (items == null) {
+      items = new Array<T>();
```

Comment:
In JS/TS when you doing `[1,2].concat(null)` result is `[1,2,null]`. But for AS for this case need new conversion. I see two variants whrn items equal to null:
1). Create new array with `len + 1`, copy existing elements to new array and fill last element by zero, so this produce `[1,2,0]` as result.
2). Create new array with `len`, just copy whole existing elements and return `[1,2]` but as cloned.

in both variants we don't need create and link 2 arrays, just only one.

Reply:
I think the second variant would be better, when the concrete generic type is class.
So I choose the second variant to enhancement.